src/cargo.rs

```diff
@@ -26,7 +26,7 @@ impl Subcommand {
             _ => true,
         }
     }
-
+    
```

Comment:
Please fix the trailing whitespace everywhere.

Reply:
fix

Reply:
The whitespace is still there, please remove it.